awx/ui/src/components/Lookup/HostFilterLookup.js

```diff
@@ -419,6 +419,7 @@ function HostFilterLookup({
             headerRow={
               <HeaderRow qsConfig={QS_CONFIG} isSelectable={false}>
                 <HeaderCell sortKey="name">{t`Name`}</HeaderCell>
+                <HeaderCell sortKey="description">{t`Description`}</HeaderCell>
```

Comment:
When a user sorts by description does it just sort alphabetically by the first letter in the description?

Reply:
Exactly